multiauth.sql

```diff
@@ -0,0 +1,30 @@
+-- TODO: move to some place else
+
+CREATE TABLE IF NOT EXISTS fastn_user (
+    id UUID PRIMARY KEY,
+    email VARCHAR(254) NOT NULL,
+    username TEXT,
+    password TEXT,
+    name TEXT NOT NULL,
+    created_on TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS fastn_session (
+    id UUID PRIMARY KEY,
+    user_id UUID REFERENCES fastn_user(id),
+    created_on TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS fastn_oauthtoken (
+    id UUID PRIMARY KEY,
+    session_id UUID NOT NULL REFERENCES fastn_session(id) ON DELETE CASCADE,
+    token text NOT NULL,
+    provider text NOT NULL,
+    created_on TIMESTAMP DEFAULT now()
```

Comment:
This should also contain `updated_on`.

Do we also have to store refresh token? And the token validity period?

Reply:
the `updated_at` field had been added.

The current github oauth flow does not receive refresh_tokens. I'll change this to get refresh_token and store it in `fastn_oauthtoken` table along with logic to refresh tokens in another PR